tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ESNext",
```

Comment:
Otherwise an error throws with `es5`:
`Transforming object literal extensions to the configured target environment ("es5") is not supported yet`

Reply:
Won't this affect our browser support? if not sure, I'd recommend doing this in a separate PR to run tests accordingly.

Reply:
According to the CanIUse site, ES6 shows 98% support.

https://caniuse.com/?search=ECMASCRIPT

Reply:
I failed to clarify that this errors throws on instances of using `cssVar()` in component themes and attempting to compile dynamic keys... say:
```ts
[$colorBg.variable]: 'some value'
```
Which is an ES6 only feature. Not sure if this is a Chakra CLI issue (doing testing in the Chakra package is not recreating the error), and really a typescript issue within this project.